build/Dockerfile.debug

```diff
@@ -2,4 +2,4 @@ FROM turbonomic/kubeturbo:6.4.0
 COPY kubeturbo.debug /opt/turbonomic/bin/kubeturbo
 ADD dlv /usr/local/bin/dlv
 EXPOSE 40000
-ENTRYPOINT ["/usr/local/bin/dlv", "--listen=:40000", "--headless=true", "--api-version=2", "exec", "/opt/turbonomic/bin/kubeturbo", "--", "--turboconfig=/etc/kubeturbo/turbo.config", "--v=2", "--kubelet-https=true", "--kubelet-port=10250"]
+ENTRYPOINT ["/usr/local/bin/dlv", "--listen=:40000", "--headless=true", "--api-version=2", "exec", "/opt/turbonomic/bin/kubeturbo"]
```

Comment:
Why removing those options?

Reply:
because, the configmap feeds and and with this, I end up with duplicates
we should use the configmap, instead of hardcoding the entrypoint

ps: I was having problems on one of my clusters which was running the kubelet without https and I could not test it with the hardcoded options